apps/src/code-studio/pd/teacher_application/teacher_application.jsx

```diff
@@ -673,7 +674,10 @@ const TeacherApplication = React.createClass({
           By clicking “Complete and Send,” you are agreeing to allow Code.org to share the information provided in this
           survey with your assigned Regional Partner and your school district.
         </label>
-        <Button id="submitButton" onClick={this.onSubmitButtonClick}>
+        <Button
+          onClick={this.onSubmitButtonClick}
+
```

Comment:
I'd expect something like `disabled={this.state.submitting}` here. As it stands, it appears the button will not be disabled. Am I missing something?